usr/share/rear/prep/DUPLICITY/Debian/100_debian_lib.sh

```diff
@@ -0,0 +1,5 @@
+#librsync may be in different Directories on Debian, depending on the Architecture
+LIBS=(
+"${LIBS[@]}"
+$(find /usr/lib /usr/lib64 -name librsync.so.1 || LogPrint "Warning: librsync.so.1 not found! Restore may not work!")
```

Comment:
@Lukey3332 Wouldn't it better to Error instead of LogPrint as recovery will not work?